### PR TITLE
Bootstrap Release 1.21 directory

### PR DESCRIPTION
**Pull request by palnabarun**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

This PR bootstraps the release directory for Kubernetes 1.21

Follow up:
- Update the TBCs with the actual dates

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None

/area release-team
/sig release
/assign @justaugustus @saschagrunert @jeremyrickard @hasheddan @alejandrox1
/cc @kikisdeliveryservice @savitharaghunathan @bai
/cc @annajung @divya-mohan0209 @erismaster @reylejano @thejoycekung @wilsonehusin

### CODE REVIEW

File: releases/release-1.21/README.md
Comment (hasheddan) [inline releases/release-1.21/README.md:63]:
```suggestion
| **Begin [Burndown]** (MWF meetings) | Lead | Mon March 1 | week 8 | [1.21-blocking], [master-blocking], [master-informing] |
```

File: releases/release-1.21/release_team.md
Comment (reylejano) [inline releases/release-1.21/release_team.md:9]:
```suggestion
| Docs | Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`) | |
```

File: releases/release-1.21/release_team.md
Comment (reylejano) [inline releases/release-1.21/release_team.md:16]:
```suggestion
The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.21, once the 1.21 release cycle concludes.
```

**Review by reylejano — CHANGES_REQUESTED** [review]
In release_team.md:
- updated my Slack handle 
- updated the patch release version to 1.21

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

**Comment by wilsonehusin** [comment]
/lgtm

(let's get this team!) :tada:

**Review by justaugustus — CHANGES_REQUESTED** [review]
Microscopic nit: Can we use hyphens for the filenames moving forward?

So `s/releases/release-1.21/release_team.md/releases/release-1.21/release-team.md`.

**Comment by justaugustus** [comment]
(that was weird; bot interpreted my "Request Changes" as an "Approve")

**Comment by palnabarun** [comment]
That was really weird. I am opening a follow on PR for the same.

**Comment by justaugustus** [comment]
Thanks!

**Comment by savitharaghunathan** [comment]
> (that was weird; bot interpreted my "Request Changes" as an "Approve")

Could be it be because the OWNERS PR got merged. and all our lgtm's has magical implicit approval powers? So when Stephen commented on the review, the bot did something to add approval based on the previous lgtm?

**Comment by palnabarun** [comment]
@justaugustus -- follow-up https://github.com/kubernetes/sig-release/pull/1408

@savitharaghunathan -- highly likely. I was also guessing on similar lines.

**Comment by justaugustus** [comment]
> Could be it be because the OWNERS PR got merged. and all our lgtm's has magical implicit approval powers? So when Stephen commented on the review, the bot did something to add approval based on the previous lgtm?

Yep, I missed https://github.com/kubernetes/sig-release/pull/1407, but that explains it.
Nabarun got implicit approve on the `releases/` directory once that happened. Thanks y'all!

**Comment by jeremyrickard** [comment]
LOL this happened to me in 1.20 😜

**Pull request merged**
